src/app-logic/url-handling.js

```diff
@@ -581,9 +583,14 @@ export function stateFromLocation(
     transforms[selectedThreadsKey] = parseTransforms(query.transforms);
   }
 
-  let tabID = null;
+  let oldTabID = null;
   if (query.ctxId && Number.isInteger(Number(query.ctxId))) {
-    tabID = Number(query.ctxId);
+    oldTabID = Number(query.ctxId);
+  }
```

Comment:
a comment to explain the difference between both tab ids could be useful

Reply:
Yeah, added one. I will remove this in the end but still keeping while we have the active tab view. But I'm thinking about at which point I should remove the old active tab view. Maybe it's better to do it sooner than later.